src/clients/cache/momento/commands/item_get_type.rs

```diff
@@ -8,11 +9,29 @@ pub async fn item_get_type(
 ) -> std::result::Result<(), ResponseError> {
     ITEM_GET_TYPE.increment();
 
-    let result = timeout(
+    match timeout(
         config.client().unwrap().request_timeout(),
         client.item_get_type(cache_name, (*request.key).to_owned()),
     )
-    .await;
-
-    record_result!(result, ITEM_GET_TYPE)
+    .await
```

Comment:
I am not very familiar with this code, but I believe the `record_result!` macro already handles this. Were you not seeing the metrics?